packages/kit/src/core/config/index.spec.js

```diff
@@ -72,6 +72,7 @@ const get_defaults = (prefix = '') => ({
 		embedded: false,
 		env: {
 			dir: process.cwd(),
+			publicModule: '_env.js',
```

Comment:
If I'm understanding correclty this means people cannot have a `_env.js` route in their app anymore, right? Doubt that there's a conflict, but maybe there's a name that makes it even less likely?
Also, is this something we may want to use later in other places? I can't image something right now, but wanted to note that we now have a mechanism to transport other stuff across the wire this way now.

Reply:
It does mean that, though I'm not too worried about conflicts (a conflict with `_app` is more likely and that's never arisen). We had talked about a dynamic endpoint for data and stuff like that, yeah. I suppose one thing we could do is reuse `_app`, and that way we don't need new config. The only thing is we'd then have both `${base}/_app` and `${assets}/_app` in the rare case that `assets` was specified, but that's probably fine now that I think about it?